packages/@o3r/apis-manager/schematics/ng-add/index.ts

```diff
@@ -1,5 +1,5 @@
 import { chain, noop, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
-import { createSchematicWithMetricsIfInstalled } from '@o3r/schematics';
+import { createSchematicWithMetricsIfInstalled, getPackageInstallConfig } from '@o3r/schematics';
```

Comment:
I don't so, why do you think it should be?

Reply:
I am asking because in all `ng-add` schematics we are dynamically importing from `@o3r/schematics` to output an error if the package is missing.
```typescript
 try {
      const { ngAddPackages, getO3rPeerDeps, applyEsLintFix, getWorkspaceConfig, getProjectNewDependenciesType } = await import('@o3r/schematics');
      ....
    } catch (e) {
      // o3r apis-manager needs o3r/schematics as peer dep.
      context.logger.error(`[ERROR]: Adding @o3r/apis-manager has failed.
      You need to install '@o3r/schematics' to be able to use the o3r apis-manager package. Please run 'ng add @o3r/schematics' .`);
      throw (e);
    }
  };
```
In this case, I don't see the need of the dynamic import.

Reply:
Since this discussion is not blocking the PR, could you create an issue so that we can tackle it separately?